src/lib/eccentric.scss

```diff
@@ -1,44 +1,149 @@
-  */
+  --et-green-primary: #00D43B;  
+  --et-valid: #00D43B;    
+  --et-success: #00D43B;
+  --et-green-secondary: #8BFFAC;
 
-  /**
-      Author: The DesignSystems
-      LICENSE: https://www.github.com/DesignSystemsOSS/eccentrictouch/blob/master/LICENSE
-      (c) All Copyrights are reserved by The DesignSystems
-  */
+  --et-yellow-primary: #FFD028;     
+  --et-warning: #FFD028;
+  --et-yellow-secondary: #FFEA9D;
 
-  @import url('animations/animations.css');           /// animations CSS Linked
-  @import url('buttons/buttons.css');                 /// buttons CSS Linked
-  @import url('components/components.css');           /// components CSS Linked         ---- ALL THE COMPONENTS WILL BE LINKED FROM THIS LINE ----
-  @import url('contents/contents.css');               /// content CSS Linked
-  @import url('footers/footers.css');                 /// footers CSS Linked
-  @import url('effects/effects.css');                 /// effects CSS Linked
-  @import url('headers/headers.css');                 /// headers CSS Linked
-  @import url('images/images.css');                   /// images CSS Linked
-  @import url('fonts/fonts.css');                     /// fonts CSS Linked
-  /**importing root module for variables**/
+  --et-gray-primary: #CECECE;       
+  --et-gray-secondary: #D3D3D3;
+  --et-deactivated: #CECECE;
 
-  @import url('variables.css');                       /// LINKING GLOBAL VARIABLES CSS MODULE
\ No newline at end of file
+  /// fonts initialization
+  --et-serif: 'Noto Serif', serif;
+  --et-sans-serif: 'Poppins', sans-serif;
+  --et-description: 'Work Sans', sans-serif;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  :root {
+    scroll-behavior: smooth;
+  }
+}
+
+body {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1400px;
+}
+
+/// button method - primary-button, default button property
+.et-button, button, .et-primaryButton {
+  color: white;
+  background-color: var(--et-blue-primary);
+  padding: 0.6em;
+  border: none;
+  margin: 0;
+  font-family: var(--et-sans-serif);
+  border-radius: 4px;
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.et-button-danger, .et-button-invalid {
+  color: white;
+  background-color: var(--et-danger);
+  padding: 0.6em;
+  border: none;
+  margin: 0;
+  font-family: var(--et-sans-serif);
+  border-radius: 4px;
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.et-button-success, .et-button-valid {
+  color: white;
+  background-color: var(--et-green-primary);
+  padding: 0.6em;
+  border: none;
+  margin: 0;
+  font-family: var(--et-sans-serif);
+  border-radius: 4px;
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.et-button-warning {
+  color: white;
+  background-color: var(--et-yellow-primary);
+  padding: 0.6em;
+  border: none;
+  margin: 0;
+  font-family: var(--et-sans-serif);
+  border-radius: 4px;
+  font-size: inherit;
+  line-height: inherit;
+}
+h1,.et-h1,h2,.et-h2,h3,.et-h3,h4,.et-h4,h5,.et-h5,h6,.et-h6{
```

Comment:
Alright, This feature-request is done, and comes under the sprint 01 #77

Reply:
Thanks @yashsehgal